src/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.html

```diff
@@ -1,6 +1,6 @@
 <template>
 
-    <lightning-record-edit-form record-id={recordId} object-api-name="npe03__Recurring_Donation__c">
+    <lightning-record-edit-form record-id={recordId} object-api-name="npe03__Recurring_Donation__c" onload={dispatchScheduleChange}>
```

Comment:
Adding `onload` event is key to displaying Elevate credit card form when *existing* Elevate RD is being edited (W-8585272 ).